content/docs/scale/_index.md

```diff
@@ -41,39 +42,66 @@ The test was run in different OSM form factors, factoring in different amounts o
 	- Correctness is ensured. It is checked that all TrafficSplit server members are eventually reached.
 - Test timeout for network correctness: 150 seconds
 
-## Assessment and Limits
+# Assessment and Limits
 *Note: Assuming proxy per pod, so pod/proxies can be used interchangeably.*
-### CPU
-#### OSM Controller
-- **1vcpu per 500 proxies**, however giving more cpu does not scale linearly (m<1) with current architecture; horizontal scaling should be considered to increase supported mesh size. 
-- Network settlment times vary from **<10s with no pods to +2min at 700 pods**. 
+
+Test failed at around 1200 pods, with kubernetes unable to bring up in time a pod in the mesh.
```

Comment:
What does "in time" mean here exactly?

Reply:
all testing waits for resources to be present and up and running a certain amount of time. It is this time in this case.

Reply:
It may help to clarify what that threshold is here in the doc, or at least say that it's a limit we define in the test.